go/vt/vtctl/reparentutil/util.go

```diff
@@ -58,7 +58,8 @@ const (
 // cell as the current primary, and to be different from avoidPrimaryAlias. The
 // tablet with the most advanced replication position is chosen to minimize the
 // amount of time spent catching up with the current primary. Further ties are
-// broken by the durability rules.
+// broken by the durability rules. Tablets taking backups are excluded from
```

Comment:
@ejortegau I think this comment is no longer accurate

Reply:
No, it's not. It reflects the fact that PRS will not promote a tablet that is taking a backup.